stories/core/Table.stories.tsx

```diff
@@ -1194,3 +1194,6 @@ Full.args = {
   isSortable: true,
   emptyFilteredTableContent: 'No results found. Clear or try another filter.',
 };
+
+export const Condensed: Story<TableProps> = Basic.bind({});
```

Comment:
You need to approve new visual test image. https://github.com/iTwin/iTwinUI-react/blob/main/CONTRIBUTING.md#visual-testing